### ISSUE TITLE
Anyone got improved performance with this implementation?

### ISSUE THREAD

**Issue by twmht** [tiandunx/SV-X-Softmax#1]
Does Anyone get improved performance with this implementation?

**Comment by tiandunx** [comment]
you can set mask = 1.2, margin = 0.35,

**Comment by davidjimenezphd** [comment]
@tiandunx I have reimplemented SVX-loss function in tensorflow and I've made unit tests to verify the correct behaviour of it. However, when I train a Resnet50 architecture from scratch using SVX loss and MS1Mv2 dataset (the purged one) the loss diverges rapidly. Have you experienced something similar? Any recommendation?

**Comment by ykk648** [comment]
no improvement, maybe I use it in a wrong way :(

**Comment by ykk648** [comment]
@davidjimenezphd same problem

**Comment by ykk648** [comment]
@twmht got 1-2% improvement in 1e-6 privite test dataset
@tiandunx thanks, i'm trying to add the combine-arcface

**Comment by davidjimenezphd** [comment]
@cloisonne How did you manage to correct the problem?

**Comment by ykk648** [comment]
@davidjimenezphd We used different backbone, seems like different situation, as u know, arcface or other kinds of margin-based loss are hard to train at the beginning, u should tune lr, margin or other params to make the loss convergence, otherwise, try RAdam.
